src/main/webapp/view/discussion/discussion.js

```diff
@@ -45,27 +49,32 @@ async function intializeDiscussion() {
  * Posts a new comment using the main post textarea.
  */
 async function postNewComment() {
-  postAndReload(ELEMENT_POST_TEXTAREA);
+  postAndReload(
+      ELEMENT_POST_TEXTAREA, undefined, newCommentTimestampMilliseconds);
 }
 
 /**
  * Posts the content of {@code inputField} as a reply to {@code parentId}.
  */
 async function postReply(inputField, parentId) {
-  postAndReload(inputField, parentId);
+  postAndReload(inputField, parentId, undefined);
```

Comment:
nit: Can you add an inline comment for `undefined`? You can also remove the last parameter all together because you assigned a default value for `postAndReload`.

Reply:
Done